src/FRP/Event.purs

```diff
@@ -107,3 +108,11 @@ foreign import subscribe
    . Event a
   -> (a -> Eff (frp :: FRP | eff) r)
   -> Eff (frp :: FRP | eff) (Eff (frp :: FRP | eff) Unit)
+
+-- | Create an event and a function which supplies a value to that event.
+foreign import create
+  :: forall eff a
+   . Eff (frp :: FRP | eff)
```

Comment:
I wonder if we should expose an `onCancel` method here, so that users can respond to cancellations. I suspect we probably don't need it, since most events we care about "live forever" (key presses, mouse movements etc.) so let's go with this for now and come back to it later if need be.